python/frontend/cirrus/cirrus/serialization.py

```diff
@@ -0,0 +1,114 @@
+# Helper functions for S3.
```

Comment:
This file is a mix of s3 functions, LambdaThread class and serialization.
I would create a separate file for the LambdaThread. The rest you can put in a utils.py file.